plone/app/customerize/tests/testCustomizeView.txt

```diff
@@ -1,8 +1,17 @@
-This tests a roundtrip of customizing, changing and deleting a template-based
-view via the ZMI:
 
-  >>> browser = self.site_administrator_browser
+First we have to set up some things and login.
+
+  >>> app = layer['app']
+  >>> from plone.testing.z2 import Browser
+  >>> browser = Browser(app)
   >>> browser.handleErrors = True
+  >>> browser.raiseHttpErrors = True
+  >>> browser.addHeader('Authorization', 'Basic admin:secret')
```

Comment:
Unfortunately it seems this is not enough. five.customerize is not installed properly with your new test fixture. This is why this solves the test isolation problems.